include/grpc/impl/channel_arg_names.h

```diff
@@ -328,6 +328,9 @@
 /** Channel arg to set http proxy per channel. If set, the channel arg
  *  value will be preferred over the environment variable settings. */
 #define GRPC_ARG_HTTP_PROXY "grpc.http_proxy"
+/** Channel arg to set proxy per channel. If set, the channel arg
+ *  value will be preferred over the environment variable settings. */
+#define GRPC_ARG_ADDRESS_PROXY "grpc.address_proxy"
```

Comment:
Please add the second channel arg to support specifying the list of address ranges for which this proxy will be used.  I don't think we want to add one without the other.

Reply:
I'd prefer to see the second channel arg added as part of this PR.  These knobs are both for the same feature, and the feature isn't useful without both of them.  And I don't want to risk this PR getting into a release without the other one.

Reply:
Done.